<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>providerDialog</class>
 <widget class="QDialog" name="providerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/middle.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1_3">
    <attribute name="title">
     <string>供货商清单</string>
    </attribute>
    <widget class="QTableView" name="tableView_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>661</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>162</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="search_type">
        <item>
         <property name="text">
          <string>所有</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>生鲜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>百货</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="search_submit">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string> 导出到 Excel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2_3">
    <attribute name="title">
     <string>添加供货商</string>
    </attribute>
    <widget class="QPushButton" name="add_submit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_clear">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>421</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_6">
        <property name="text">
         <string>供货商号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="add_sup_id"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Label_7">
        <property name="text">
         <string>供货类型</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_8">
        <property name="text">
         <string>商品名称</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_10">
        <property name="text">
         <string>联系方式</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="add_sup_phone"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="add_sup_name"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="add_sup_type">
        <item>
         <property name="text">
          <string>生鲜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>百货</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>删除供货商</string>
    </attribute>
    <widget class="QPushButton" name="delete_clear">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_submit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认删除</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>391</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_11">
        <property name="text">
         <string>供货商号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="delete_sup_id"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_12">
        <property name="text">
         <string>供货类型</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_13">
        <property name="text">
         <string>商品名称</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label_15">
        <property name="text">
         <string>联系方式</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="request_sup_info">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="delete_sup_type">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="delete_sup_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="delete_sup_phone">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>修改供货商信息</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>441</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_14">
        <property name="text">
         <string>供货商号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="update_sup_id"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_16">
        <property name="text">
         <string>供货类型</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_17">
        <property name="text">
         <string>商品名称</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="update_sup_name"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label_18">
        <property name="text">
         <string>联系方式</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="update_sup_phone"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="request_sup_info_2">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="update_sup_type">
        <item>
         <property name="text">
          <string>生鲜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>百货</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="update_clear">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_submit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
